src/components/MultiBarButton/index.tsx

```diff
@@ -19,7 +19,8 @@ export const MultiBarButton: React.FC<Props> = ({
 
   React.useEffect(() => {
     Animated.spring(animated, {
-      toValue: extrasVisible ? 1 : 0
+      toValue: extrasVisible ? 1 : 0,
+      useNativeDriver: true
```

Comment:
ReactNative doesn't support `useNativeDriver: true` for properties like: x, y, width, height, so should be false, please fix it then I can merge.

Reply:
I saw that you took care of it. thanks :)